assets/edit.1

```diff
@@ -0,0 +1,15 @@
+.TH EDIT 1 "version 1.0" "May 2025"
+.SH NAME
+edit \- a simple text editor
+.SH SYNOPSIS
+edit [file]
+.SH DESCRIPTION
+edit is a simple text editor inspired by MS-DOS edit.
+.SH EDITING
+Edit is an interactive mode-less editor. Use Alt-F to access the menus.
+.SH OPTIONS
+Pass the name of the file to edit.
```

Comment:
```suggestion
.SH ARGUMENTS
.TP
\fIFILE[:LINE[:COLUMN]]\fP
The file to open, optionally with line and column (e.g., \fBfoo.txt:123:45\fP).
.SH OPTIONS
.TP
\fB\-h\fP, \fB\-\-help\fP
Print the help message.
.TP
\fB\-v\fP, \fB\-\-version\fP
Print the version number.
```

Reply:
https://github.com/microsoft/edit/blob/2b65e00952ee6bd672b933f2dac32e226d343721/src/bin/edit/main.rs#L288-L294

Reply:
@ivanovkirilg The suggested synopsis section above already indicates that `edit` can accept multiple positional arguments (`\fBedit\fP [\fIOPTIONS\fP]... [\fIFILE\fP]...`). So I don't think it's necessary to indicate again in the arguments section that `edit` can take multiple arguments. I think `[FILE[:LINE[:COLUMN]] ...]` is a bit complicated.